README.md

```diff
@@ -47,16 +47,16 @@ The Swift Protocol is an implementation of a Decentralized Autonomous Organizati
 
 ## Account Types
-The Swift Protocol functions as a DAO that allows for real world governance to interconnect with the blockchain. Accounts have the ability to vote on certain events that affect how the Swift Protcol functions. This allows the Swift Protocol to react naturally to real world events by remaining in a fluid state. The Swift Protocol is decentralized to avoid single points of failure while still enabling identity verification and financial controls.
+The Swift Protocol functions as a DAO that allows for real-world governance to interconnect with the blockchain. Accounts have the ability to vote on certain events that affect how the Swift Protcol functions. This allows the Swift Protocol to react naturally to real-world events by remaining in a fluid state. The Swift Protocol is decentralized to avoid single points of failure while still enabling identity verification and financial controls.
 
 ### Swift Citizens
-Swift Citizens are unique inidividuals that have been validated by an Idenity Provider. Citizens have the ability to add claim to the blockchain once every day (days begin and end at midnight UTC). Swifts will be awarded based on the amount of days passed with a maximum of 7. For example, if the 3 days have passed and a user makes a claim with the current production rate resting at 100 Swifts. 300 Swifts will be awarded.
+Swift Citizens are unique inidividuals that have been validated by an [Identity Provider](#identity-providers). Citizens have the ability to add claims to the blockchain once every day (days begin and end at midnight UTC). Swifts will be awarded based on the number of days passed--with a maximum of seven. For example, if the three days have passed and a user makes a claim with the current production rate at 100 Swifts, 300 Swifts will be awarded.
 
 ### Swift Accounts
-Swift Accounts are not connected to any particularly real world identity and therefore do not receive Basic Income. These accounts still must be approved by Identity Providers before being created. An Identity Provider must be linked to all Swift Accounts, however Identity Providers are not allowed to make transactions on behalf of a Swift Account. Delegated Nodes and Identity Providers both inherit from the Swift Account type.
+Swift Accounts are not connected to any particularly real-world identity and therefore do not receive Basic Income. These accounts still must be approved by Identity Providers before being created. An Identity Provider must be linked to all Swift Accounts, however Identity Providers are not allowed to make transactions on behalf of a Swift Account. Delegated Nodes and Identity Providers both inherit from the Swift Account type.
```

Comment:
I have another mistake here! It should be `particular` instead of `particularly`

Reply:
Yes I saw that, but I'm not going to touch the next few paragraphs until you fix the "Swift Accounts" naming confusion.